src/@next/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { Button, PrimaryButton } from '../Button';
+import { Neutral } from '../utilities/colors';
+import {
+  EmptyStateContainer,
+  EmptyStateContentContainer,
+  StyledButtonGroup,
+  StyledDescription,
+  StyledHelpText,
+  StyledImage,
+  StyledTitle,
+} from './EmptyStateStyle';
+
+export const imageName = ['empty-carton', 'empty-mailbox', 'safety-cone'];
```

Comment:
@ninariccimarie we may have to narrow down the type here as currently we can pass any string
```

export const imageName = ['empty-carton', 'empty-mailbox', 'safety-cone'] as const; 

```

Reply:
@Afsal102 I'm already using these as the type: https://github.com/glints-dev/glints-aries/pull/744/files#diff-7e72d5b4c46257b14b04072cd667efebe24feda83031d0275c67fcf50ff4fb6cR26
let me throw a warning if it's not found so JS can see the error since only TS files can throw the error for the typing

Reply:
@ninariccimarie am aware of it, but even with this right u can pass any string value(not strict), but if u narrow the type down, the user will be prompted to select one of the values in `imageName` (type safety)

Reply:
@Afsal102 what do you mean by narrow the type down?

Reply:
> @ninariccimarie am aware of it, but even with this right u can pass any string value(not strict), but if u narrow the type down(readonly), the user will be prompted to select one of the values in `imageName` (type safety)

Reply:
addressed @Afsal102